src/basic/virt.c

```diff
@@ -236,9 +275,13 @@ static int detect_vm_uml(void) {
         r = read_full_file("/proc/cpuinfo", &cpuinfo_contents, NULL);
         if (r < 0)
                 return r;
-        if (strstr(cpuinfo_contents, "\nvendor_id\t: User Mode Linux\n"))
+
+        if (strstr(cpuinfo_contents, "\nvendor_id\t: User Mode Linux\n")) {
+                log_debug("UML virtualization found in /proc/cpuinfo (%s)", cpuinfo_contents);
                 return VIRTUALIZATION_UML;
+        }
 
+        log_debug("No virtualization found in /proc/cpuinfo (%s)", cpuinfo_contents);
```

Comment:
In this case I was thinking in english only names are spelled in uppercase?

log_debug("No virtualization found in /proc/cpuinfo (%s)", cpuinfo_contents);

 

Sorry, I am german, no native speaker ;=(

 

 

Von: Zbigniew Jędrzejewski-Szmek [mailto:notifications@github.com] 
Gesendet: Dienstag, 23. Februar 2016 20:39
An: systemd/systemd <systemd@noreply.github.com>
Cc: nafets227 <infos@nafets.de>
Betreff: Re: [systemd] Bugfix set virt=none when running on XEN Dom0 (#2639)

 

In src/basic/virt.c <https://github.com/systemd/systemd/pull/2639#discussion_r53836796> :

>  
> +        log_debug("No virtualization found in /proc/cpuinfo (%s)", cpuinfo_contents);

Ditto.

—
Reply to this email directly or view it on GitHub <https://github.com/systemd/systemd/pull/2639/files#r53836796> .  <https://github.com/notifications/beacon/AFW-poUZUkD091jCZgSjfXUqnEoyngCJks5pnKzpgaJpZM4Hbhav.gif>